packages/internal/test-util/src/lib/error-throwing-driver/error-throwing-driver.module.ts

```diff
@@ -22,7 +23,13 @@ export class ErrorThrowingDriverModule {
 
     return {
       ngModule: ErrorThrowingDriverRootModule,
-      providers: [{ provide: errorThrowingDriverConfigToken, useValue: fullConfig }],
+      providers: [
+        {
+          provide: errorThrowingDriverConfigToken,
+          useFactory: (logDriverConfig: LumberjackLogDriverConfig) => ({ ...logDriverConfig, ...fullConfig }),
```

Comment:
suggestion: add explicit return type to avoid type errors.
```suggestion
          useFactory: (logDriverConfig: LumberjackLogDriverConfig): ErrorThrowingDriverConfig => ({ ...logDriverConfig, ...fullConfig }),
```